### ISSUE TITLE
tableName in nest struct 

### ISSUE THREAD

**Issue by kongfei605** [go-pg/pg#432]
```go
App struct{
tableName struct{} `sql:"my.app"`
Name string
...
}

(app *App)Insert() (err error){
_,err:= db.Model(app).Insert()
....
return
}


AppExport struct{
tableName struct{} `sql:"my.appext"`
models.App
}



(aex *AppExport) Insert() (err error){
_,err:= another_db.Model(aex).Insert()
....
return 
}
```

there are two db connections,  AppExport's Insert Method will cover App's Insert Method. 
db.Model(&AppExport{}) will  share the same tableName with db.Model(&App{}).

I hope I could export data from one table to another table without the same table name.  This Model method may need to be  restructured.

**Comment by vmihailenco** [comment]
I tried to write a [test](https://github.com/go-pg/pg/pull/433) for this behaviour and it works as expected. Can you make sure you are using the latest version?

**Comment by kongfei605** [comment]
thanks to your kind reply @vmihailenco , I tryied again and it did not work for me. It works fine with pg.v4, but pg.v4 could find column when *struct is embeded.

**Comment by vmihailenco** [comment]
@kongfei605 can you please write reproducible example?

**Comment by kongfei605** [comment]
I will write one example later and feed back, please don't close this issue
now.

2016-12-13 17:00 GMT+08:00 Vladimir Mihailenco <notifications@github.com>:

**Comment by kongfei605** [comment]
I write one example for insert test: https://github.com/kongfei605/pgtest
someone may help me figure out the problem

**Comment by vmihailenco** [comment]
Thanks, fixed in v5.1.9
